pkgs/servers/nosql/influxdb2/default.nix

```diff
@@ -0,0 +1,125 @@
+{ buildGoModule
+, buildGoPackage
+, fetchFromGitHub
+, git
+, go-bindata
+, lib
+, llvmPackages
+, mkYarnPackage
+, pkg-config
+, rustPlatform
+, ... }:
+let
+  version = "2.0.2";
+  libflux_version = "0.95.0";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "influxdata";
+    repo = "influxdb";
+    sha256 = "1vv0f5zpxl7s6rsadhsj25zxk04m7a0jzghrbzynjv7xv7ml90r7";
+    # The build process requires `git rev-parse HEAD` to work, so we
+    # ask for submodules (to force the use of fetchGit) and tell it to
+    # leave .git in place. A shallow clone is fine though.
+    fetchSubmodules = true;
+    leaveDotGit = true;
+  };
+
+  ui = mkYarnPackage {
+    src = src;
+    packageJSON = "${src}/ui/package.json";
+    yarnLock = "${src}/ui/yarn.lock";
+    extraBuildInputs = [ git ];
+    configurePhase = ''
+      cp -r $node_modules ui/node_modules
+      rsync -r $node_modules/../deps/chronograf-ui/node_modules/ ui/node_modules
```

Comment:
Could we use symlinks here?

Reply:
I had some issues where yarn was angry when I put symlinks in node_modules, but I was having other problems as well. I'll see if I can avoid copying all this stuff around.

Reply:
Tried to do stuff with symlinks, but for some reason that results in webpack being unable to find any packages... including its own CSS parser, so the compile fails hilariously.